modules/woo-commerce/qwc-admin.php

```diff
@@ -26,27 +34,17 @@ function qtranxf_wc_add_filters_admin() {
 
     // not all combinations are in use, but it is ok, they may be added in the future.
     foreach ( $email_ids as $name => $priority ) {
-        add_filter( 'woocommerce_email_recipient_' . $name, 'qtranxf_useCurrentLanguageIfNotFoundUseDefaultLanguage', $priority );
-        add_filter( 'woocommerce_email_subject_' . $name, 'qtranxf_useCurrentLanguageIfNotFoundUseDefaultLanguage', $priority );
-        add_filter( 'woocommerce_email_heading_' . $name, 'qtranxf_useCurrentLanguageIfNotFoundUseDefaultLanguage', $priority );
-        add_filter( 'woocommerce_email_content_' . $name, 'qtranxf_useCurrentLanguageIfNotFoundUseDefaultLanguage', $priority );
-    }
-
-    $email_common = array(
-        'woocommerce_email_footer_text'  => 20,
-        'woocommerce_email_from_address' => 20,
-        'woocommerce_email_from_name'    => 20,
-    );
-
-    foreach ( $email_common as $name => $priority ) {
-        add_filter( $name, 'qtranxf_useCurrentLanguageIfNotFoundUseDefaultLanguage', $priority );
+        $admin_hooks[ 'woocommerce_email_recipient_' . $name ] = $priority;
+        $admin_hooks[ 'woocommerce_email_subject_' . $name ]   = $priority;
+        $admin_hooks[ 'woocommerce_email_heading_' . $name ]   = $priority;
+        $admin_hooks[ 'woocommerce_email_content_' . $name ]   = $priority;
     }
 
-    add_filter( 'woocommerce_attribute_taxonomies', 'qtranxf_useCurrentLanguageIfNotFoundUseDefaultLanguage' );
-    add_filter( 'woocommerce_variation_option_name', 'qtranxf_useCurrentLanguageIfNotFoundUseDefaultLanguage' );
+    qtranxf_add_filters( [ 'text' => $admin_hooks ] );
 }
 
 qtranxf_wc_add_filters_admin();
+global $wp_filter;
```

Comment:
This doesn't seem to be used.

Reply:
forgotten there... https://github.com/qtranslate/qtranslate-xt/pull/1194/commits/6fe1816eefd0847ec0ec174755df87574143495b